.travis.yml

```diff
@@ -5,6 +5,15 @@ language: node_js
 cache:
   directories:
   - node_modules
+before_install:
+- |
+    if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
```

Comment:
It looks like none of that machinery should be required any more: https://github.com/facebook/react/commit/08b1515f7fb82c64b3f218bf60964b6b75fa6be6